scripts/kconfig/hardened.csv

```diff
@@ -74,6 +74,7 @@ BT_PASSKEY_KEYPRESS,n,experimental
 BT_PER_ADV_RSP,n,experimental
 BT_PER_ADV_SYNC_RSP,n,experimental
 BT_RFCOMM,n,experimental
+BT_SPP,n,experimental
```

Comment:
Please rebase. This section doesn't exist anymore as experimental Kconfigs are now auto detected

Reply:
got it，next patch would fix it